rero_ils/modules/locations/api.py

```diff
@@ -69,20 +70,24 @@ def extended_validation(self, **kwargs):
         return True
 
     @classmethod
-    def get_pickup_location_pids(cls, patron_pid=None):
+    def get_pickup_location_pids(cls, patron_pid=None, item_barcode=None):
```

Comment:
why not passing the item itself? This will avoid to retrieve the item again (1 es + 1 db query), or at lease give the item_pid instead (1 db query).

Reply:
change to use `item_pid` instead of `item_barcode`.